src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsService.kt

```diff
@@ -52,8 +52,8 @@ class CourtCaseEventsService(
     }
   }
 
-  private fun findByFirstNameSurname(firstName: String? = "firstName", surname: String? = "surname"): List<DefendantEntity> {
-    return defendantRepository.findAllByFirstNameAndSurname(firstName!!, surname!!)
+  private fun findByFirstNameSurname(firstName: String?, surname: String?): List<DefendantEntity> {
+    return defendantRepository.findAllByFirstNameAndSurname(firstName, surname!!)
```

Comment:
Surname doesn't need to be force unwrapped anymore? can remove `!!`. Otherwise that will cause a null pointer exception as well

Reply:
I need a rest!